sdks/wasm/framework/src/WebAssembly.Net.Http/WebAssemblyHttpRequestMessageExtensions.cs

```diff
@@ -0,0 +1,199 @@
+        }
+
+        /// <summary>
+        /// Configures a value for the 'integrity' option for the HTTP request.
+        /// </summary>
+        /// <param name="requestMessage">The <see cref="HttpRequestMessage"/>.</param>
+        /// <param name="integrity">The subresource integrity descriptor.</param>
+        /// <returns>The <see cref="HttpRequestMessage"/>.</returns>
+        public static HttpRequestMessage SetIntegrity(this HttpRequestMessage requestMessage, string integrity)
+            => SetFetchOption(requestMessage, "integrity", integrity);
+
+        /// <summary>
+        /// Configures a value for the HTTP request.
+        /// </summary>
+        /// <param name="requestMessage">The <see cref="HttpRequestMessage"/>.</param>
+        /// <param name="name">The name of the HTTP fetch option.</param>
+        /// <param name="value">The value, which must be JSON-serializable.</param>
+        /// <returns>The <see cref="HttpRequestMessage"/>.</returns>
+        /// <remarks>
+        /// See https://developer.mozilla.org/en-US/docs/Web/API/WindowOrWorkerGlobalScope/fetch
+        /// </remarks>
+        public static HttpRequestMessage SetFetchOption(this HttpRequestMessage requestMessage, string name, object value)
+        {
+            if (requestMessage is null)
+            {
+                throw new ArgumentNullException(nameof(requestMessage));
+            }
+
+            const string FetchRequestOptionsKey = "FetchRequestOptions";
+            IDictionary<string, object> fetchOptions;
+
+            if (requestMessage.Properties.TryGetValue(FetchRequestOptionsKey, out var entry))
+            {
+                fetchOptions = (IDictionary<string, object>)entry;
+            }
+            else
+            {
+                fetchOptions = new Dictionary<string, object>();
+                requestMessage.Properties[FetchRequestOptionsKey] = fetchOptions;
+            }
+
+            fetchOptions[name] = value;
+
+            return requestMessage;
+        }
+
+        /// <summary>
+        /// Configures streaming response for the HTTP request.
+        /// </summary>
+        /// <param name="requestMessage">The <see cref="HttpRequestMessage"/>.</param>
+        /// <param name="streamingEnabled"><see langword="true"> if streaming is enabled; otherwise false.</param>
+        /// <returns>The <see cref="HttpRequestMessage"/>.</returns>
+        /// <remarks>
+        /// This API is only effective when the browser HTTP Fetch supports streaming.
+        /// See https://developer.mozilla.org/en-US/docs/Web/API/ReadableStream.
+        /// </remarks>
+        public static HttpRequestMessage SetStreamingEnabled(this HttpRequestMessage requestMessage, bool streamingEnabled)
+        {
+            if (requestMessage is null)
+            {
+                throw new ArgumentNullException(nameof(requestMessage));
+            }
+
+            requestMessage.Properties["StreamingEnabled"] = streamingEnabled;
```

Comment:
In gRPC we're going to be cutting a new branch for the next release soon - https://github.com/grpc/grpc-dotnet/pull/796

I'd like to include setting the property to enable streaming in WASM in. It won't do anything for now, but once Blazor supports opting in to streaming gRPC will start working without a new update.

Is `StreamingEnabled` with a `bool` what you're going to use here?